largescale/run-million-collection-test.sh

```diff
@@ -0,0 +1,105 @@
+#! /usr/bin/env bash
+# 
+# A script to prepare test environment and execute million collections testing.
+# 
+
+set -eux
+
+MONGO_REPO="https://github.com/mongodb/mongo"
+MONGO_TESTS_REPO="https://github.com/wiredtiger/mongo-tests"
+POCDRIVER_REPO="https://github.com/wiredtiger/POCDriver"
+POCDRIVER_BRANCH="mongodb-million-collections"
+TEST_DIR="mongo-million-collection-test"
+PERF_MAKE_FLAGS="-j 20"
+
+function prepare_test_env() { 
+	# Backup the old test directory if found
+	if [ -d ${TEST_DIR} ]; then
+		mv ${TEST_DIR} ${TEST_DIR}_$(date +%F-%H:%M)
+	fi
+
+	# Clone the mongo repo as the top test directory
+	git clone ${MONGO_REPO} ${TEST_DIR} || exit $?
+
+	# Clone other repos inside the test directory
+	cd ${TEST_DIR}
+	git clone ${MONGO_TESTS_REPO} || exit $?
+	git clone -b ${POCDRIVER_BRANCH} ${POCDRIVER_REPO} || exit $?
+	# Build from the POCDriver repo
+	( cd POCDriver && mvn clean package )
+
+	# Setup directory to record test output
+	if [ -d "results" ]; then
+		rm -rf results
+	fi
+	mkdir results
+}
+
+function merge_wiredtiger_develop() { 
+	(cd src/third_party
+	curl -OJL https://api.github.com/repos/wiredtiger/wiredtiger/tarball/develop
+	tarball=$(echo wiredtiger-wiredtiger-*.tar.gz)
+	test -f "${tarball}"
+	mkdir -p wiredtiger
+	(cd wiredtiger;
+	 rm -rf *;
+	 git checkout -- .gitignore 'SCons*';
+	 git checkout -- 'build_*/wiredtiger_config.h')
+	tar -x --strip-components 1 \
+	    --exclude '*/api' --exclude '*/dist/package' --exclude '*/examples' \
+	    --exclude '*/src/docs' --exclude '*/test' \
+	    --exclude '*/tools/wtperf_stats' \
+	    --exclude '*/tools/wtstats/template' \
+	    -C wiredtiger -f ${tarball}
+	)
+}
+
+function build_mongo() { 
+	# Modify the build scripts
+	sed -i.bak 's/ggdb/gdwarf-2/' ./SConstruct
+	sed -i 's/stdint\.h>/stdint.h>\n        #include <stdbool.h>/g' src/third_party/wiredtiger/SConscript
```

Comment:
Are these still necessary?

Reply:
These lines are copy-pasted from the Jenkins job. Will give it a try by removing the SConscript customisations.

Reply:
Commented out the 2 `sed` lines and ran the test again on `nimer`, mongod was built successfully and the testing is running normally. It looks not necessary at more, will go ahead remove them in this script, and the Jenkins job.

Reply:
Thanks Luke